### PR TITLE
Add cterm support

### PR DESCRIPTION
**Pull request by gko**
`vim` in macOS (8.1 anyway) doesn't support `termguicolors` (even in iterm2).

This is a draft of proposal on how to add a support for cterm colors here.
I've generated the closest colors with help of this https://gist.github.com/mhulse/b11e568260fb8c3aa2a8
bash script

### CODE REVIEW

File: colors/material.vim [hunk 1]
```diff
@@ -414,5 +414,83 @@ if has('nvim')
   let g:terminal_color_13 = g:terminal_color_5
   let g:terminal_color_14 = g:terminal_color_6
   let g:terminal_color_15 = g:terminal_color_7
+elseif has('terminal')
+    if g:material_theme_style == 'palenight'
+        let g:terminal_ansi_colors = [
+                    \ "#292D3E",
```
Comment (kaicataldo) [inline colors/material.vim:420]: Can we use the colors defined in the colors map so that these colors are defined in one place?

**Review by kaicataldo — CHANGES_REQUESTED** [review]
Thanks for working on this! I’m supportive of adding this :+1:

Sorry to ask this again, but do you mind including a screenshot of each theme? It allows others to provide input without having to download the branch manually and is nice for posterity.

File: colors/material.vim (hunk 1 above)
Comment (kaicataldo) [reply colors/material.vim:420]: Once this gets fixed this we should be able to land this :)

**Comment by gko** [comment]
I accidentally deleted the fork and couldn't restore it, so I recreated the PR here: #45

**Pull request closed without merging**